TESTS/mbedmicro-rtos-mbed/heap_and_stack/main.cpp

```diff
@@ -178,7 +178,7 @@ void test_heap_in_range(void)
  */
 void test_main_stack_in_range(void)
 {
-    os_thread_t *thread = (os_thread_t*) osThreadGetId();
+    mbed_rtos_storage_thread_t *thread = (mbed_rtos_storage_thread_t*) osThreadGetId();
```

Comment:
mbed_rtos_storage_*_t are just blocks of memory to backs the object. I wouldn't like to use them in different context.